app/src/context/DocumentStore.tsx

```diff
@@ -112,19 +117,52 @@ export const DocumentProvider: React.FC<{
     setFiles(updatedFiles);
   };
 
-  const reset = () => {
+  const resetFiles = () => {
     fileLocalStorage.clear();
     setFiles([]);
   };
 
+  const addDocument = async (_document: Document): Promise<string> => {
+    const id = uuid.v4() as string;
```

Comment:
I'm not sure what this guy is for? (uuid.v4)

Reply:
futureproofing for future functionality so document unique identifier isn't tied to document type